api/src/main/resources/workbench.yaml

```diff
@@ -787,7 +787,7 @@ definitions:
   WorkspaceAccessLevel:
     type: string
     description: levels of access to workspace
-    enum: [reader, writer, owner]
+    enum: [NO ACCESS, READER, WRITER, OWNER]
```

Comment:
If we're going to include NO_ACCESS here we should indicate this is equivalent to removing a user from the ACL in the description.